components/list/list.html

```diff
@@ -0,0 +1,183 @@
+    <meta name="author" content="Anupam Srivastava">
+    <title>Decora</title>
+    <link rel="stylesheet" href="../../css/all.css">
+    <script src="https://kit.fontawesome.com/7966d641ca.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <div class="container">
+        <nav class="header">
+            <a href="../../index.html" class="main-heading">
+                Decora
+            </a>
+            <div class="side-heading">
+                Docs
+            </div>
+        </nav>
+        <div class="contents">
+            <nav class="side-nav">
+                <ul>
+                    <li class="item">
+                        <a href="../avatar/avatar.html">Avatar</a>
+                    </li>
+                    <li class="item">
+                        <a href="../alert/alert.html">Alert</a>
+                    </li>
+                    <li class="item">
+                        <a href="../badge/badge.html">Badge</a>
+                    </li>
+                    <li class="item">
+                        <a href="../button/button.html">Button</a>
+                    </li>
+                    <li class="item">
+                        <a href="../card/card.html">Card</a>
+                    </li>
+                    <li class="item">
+                        <a href="../image/image.html">Image</a>
+                    </li>
+                    <li class="item">
+                        <a href="../input/input.html">Input</a>
+                    </li>
+                    <li class="item">
+                        <a href="./text-utils.html">Text Utils</a>
+                    </li>
+                    <li class="item">
+                        <a class="active" href="./list.html">List</a>
+                    </li>
+                    <li class="item">
+                        <a href="../navigation/navigation.html">Navigation</a>
+                    </li>
+                    <li class="item">
+                        <a href="../toast/toast.html">Toast</a>
+                    </li>
+                    <li class="item">
+                        <a href="../modal/modal.html">Modal</a>
+                    </li>
+                    <li class="item">
+                        <a href="../rating/rating.html">Rating</a>
+                    </li>
+                    <li class="item">
+                        <a href="../grid/grid.html">Grid</a>
+                    </li>
+                    <li class="item">
+                        <a href="../slider/slider.html">Slider</a>
+                    </li>
+                </ul>
+            </nav>
+            <div class="main-component">
+                <h1 class="comp-heading">List</h1>
+                <hr>
+                <p class="comp-desc">
+                    A list can be used to show multiple related options or links. Lists are less time taking to study as
+                    compared to paragraph.
```

Comment:
Class name should be more descriptive. For eg. 'comp-description' instead of 'comp-desc' would be much easy to understand.